src/components/like/LikeButton.js

```diff
@@ -16,7 +16,7 @@ import { getLikePost } from '../../redux/actions/dataActions';
 /** View component for displaying an icon to like a post
  * @param {ILikeButtonComponentProps} props
  */
-const LikeButton = props => {
+export const LikeButton = props => {
```

Comment:
@Ilithor FYI I exported the view component as well here. The way it works is:
`import whatever from './likeButton'`
>redux connected default

`import { LikeButton } from './likeButton'`
> UX-only component with no redux connection